wormhole-connect/src/components/FooterNavBar.tsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import { useDispatch } from 'react-redux';
+import { makeStyles } from 'tss-react/mui';
+import { ICON } from 'utils/style';
+import { Route, setRoute } from 'store/router';
+
+const useStyles = makeStyles()(() => ({
+  menuIcon: ICON,
+  menu: {
+    display: 'flex',
+    flexDirection: 'row',
+    gap: '8px',
+    padding: '8px',
+  },
+  menuItem: {
+    padding: '16px 0',
+    textAlign: 'left',
```

Comment:
```suggestion
    textAlign: 'center',
    flex: 1,
```

<img width="822" alt="image" src="https://github.com/wormhole-foundation/wormhole-connect/assets/11276821/61805b6a-3adc-4e1b-b043-dd2edcf9e7e0">

Dismiss this comment and the previous one if this is the intended design

Reply:
we do not have a design defined yet for this, so, I'll update this with your suggestions

Reply:
applied